### PR TITLE
Add `iosArm32` support

### PR DESCRIPTION
**Pull request by Jahor**
Add an extra target platform

### CODE REVIEW

**Comment by twyatt** [comment]
Thanks so much for the contribution! I need to check with our legal team, I think we'll need a CLA (or similar) in place to accept external contributions. Shouldn't be a problem (apologies that it might slow this first external contribution a bit). I'll follow up soon.

**Comment by twyatt** [comment]
Heard back from legal, we don't need a CLA. 🎉 
I'll look over this PR today or tomorrow.
Thanks again for the contribution and patience!

File: core/src/appleMain/kotlin/NSData.kt [hunk 1]
```diff
@@ -19,6 +21,6 @@ internal fun NSData.toByteArray(): ByteArray = ByteArray(length.toInt()).apply {
 internal fun ByteArray.toNSData(): NSData = memScoped {
     NSData.create(
         bytes = allocArrayOf(this@toNSData),
-        length = size.toULong()
+        length = size.convert<NSUInteger>()
```
[the remark this answers is not in the document]
Comment (davidtaylor-juul) [reply core/src/appleMain/kotlin/NSData.kt:24]: Line 14 is Kotlin stdlib API not cinterop though.

**Review by davidtaylor-juul — APPROVED** [review]

File: core/build.gradle.kts [hunk 2]
```diff
@@ -47,6 +48,13 @@ kotlin {
             }
         }
 
+        val iosArm32Main by getting {
+            dependencies {
+                api(coroutines("core", version = "1.5.0-native-mt!!"))
```
Comment (twyatt) [inline core/build.gradle.kts:53]: Per #129, Kable no longer depends on `-native-mt` Coroutines and expects library consumers to properly pull in the `-native-mt` Coroutines for targets that need it.

```suggestion
```

File: core/src/appleMain/kotlin/NSData.kt [hunk 3]
```diff
@@ -2,10 +2,12 @@ package com.juul.kable
 
 import kotlinx.cinterop.addressOf
 import kotlinx.cinterop.allocArrayOf
+import kotlinx.cinterop.convert
 import kotlinx.cinterop.memScoped
 import kotlinx.cinterop.usePinned
 import platform.Foundation.NSData
 import platform.Foundation.create
+import platform.darwin.NSUInteger
```
Comment (twyatt) [inline core/src/appleMain/kotlin/NSData.kt:10]:
```suggestion
```

**Comment by twyatt** [comment]
Thanks so much for the contribution @Jahor!

**Comment by twyatt** [comment]
Was released in 0.7.1.

**Pull request merged**